src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -92,8 +83,13 @@ function ForWhoSection(): React.ReactElement {
               </Box>
             </Box>
             <Box sx={styles.square}>
-              <Box component="img" {...bigScreenProps} sx={styles.bigScreen} loading="lazy" />
-              <Box component="img" {...smallScreenProps} sx={styles.smallScreen} loading="lazy" />
+            <Box sx={styles.bigScreen}>
+              <Box sx={styles.bigScreen}>
+         <img alt={t('alts.big_screen')} {...getImageProps('/assets/img/about-vilna/desktop.jpg')} loading="lazy" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)